Camera/AmetekS711EuresysCoaxlinkQsfp+/camera_phantom_s711.hh

```diff
@@ -70,14 +72,39 @@ class EHoloGrabber
 
         depth_ = static_cast<PixelDepth>(gentl.imageGetBytesPerPixel(pixel_format));
 
-        for (unsigned i = 0; i < grabbers_.length(); ++i)
-            grabbers_[i]->setInteger<StreamModule>("BufferPartCount", nb_images_per_buffer_);
+        // The below loop will check which grabbers are available to use, i.e the ones which are connected to a camera
+        // We don't use Euresys::EGrabberDiscovery because it doesn't allow us to detect when a frame grabber is
+        // connected to something or not
+        for (size_t ix = 0; ix < grabbers_.length(); ++ix)
+        {
+            try
+            {
+                // Try to query the remote device (the camera)
+                grabbers_[ix]->getString<RemoteModule>("Banks");
+            }
+            catch (const Euresys::gentl_error& e)
+            {
+                continue;
+            }
```

Comment:
No need for the definition of an error variable, as it won't be used.

You can just remove the 'e'.